sdk/ai/azure-ai-projects/azure/ai/projects/telemetry/agents/_ai_agents_instrumentor.py

```diff
@@ -56,18 +56,16 @@
 )
 from azure.core import CaseInsensitiveEnumMeta  # type: ignore
```

Comment:
Please rebase. Marko already resolved errors in this folder

Reply:
done